src/fastapi_aad_auth/providers/aad.py

```diff
@@ -85,7 +85,11 @@ def _build_redirect_uri(self, request: Request):
                 port = ''
             else:
                 port = f':{request.url.port}'
-            redirect_uri = f'{request.url.scheme}://{request.url.hostname}{port}{self._redirect_path}'
+            if 'X-Forwarded-Proto' in request.headers and request.headers['X-Forwarded-Proto'] == 'https':
+                url_scheme='https'
+            else:
+                url_scheme='http'
```

Comment:
can you fix the lint here to be `url_scheme = 'http'` and the same for the https above

Reply:
sorry, busy with other projects, fixed